packages/caravan-psbt/src/psbtv2/psbtv2.test.ts

```diff
@@ -1665,3 +1684,20 @@ describe("PsbtV2.setInputSequence", () => {
     expect(emptyPsbt.isRBFSignaled).toBe(false);
   });
 });
+
+describe("PsbtV2.toV0", () => {
+  // A PSBT is still valid if it includes unrecognized key-value pairs, so we
+  // don't need to check that the incompatible keys were removed, but we do need
+  // to check that the required PSBT_GLOBAL_UNSIGNED_TX was properly created.
+  test.each(BIP_174_VECTORS_VALID_PSBT)(
+    "Properly constructs PSBT_GLOBAL_UNSIGNED_TX. $case",
+    (vect) => {
+      const psbtv2 = PsbtV2.FromV0(vect.base64, true);
+      const psbtv0 = psbtv2.toV0("hex");
+      // At a minimum, the raw tx cannot start before string index 16 because
+      // the hex psbt must at least start with "70736274ff0100" and one byte for
+      // length.
+      expect(psbtv0.indexOf(vect.unsignedRawTx)).toBeGreaterThanOrEqual(16);
+    },
+  );
+});
```

Comment:
I started with a test that tried converting to psbtv2 and then back to psbtv0 and comparing the entire serialized psbt. I knew this would be brittle since the maps don't have to be in the same order. It worked mostly, but failed on a couple of the comparisons since mapped values were out of order.

After thinking about it, I came to the conclusion that is described in the first comment. However, a bug in this change, namely the lack of setting of locktime by `FromV0`, was revealed from the full psbt comparison tests.